README.md

```diff
@@ -34,12 +38,14 @@ Place the binary into `../../bin/polkadot-0.9.18-sudo`
 
 ```bash
 # We need to build it from source. The one from the yarn registry does not work with our code.
+cd ..
```

Comment:
Dito

Reply:
dito. here it is actually a specific requirement that the repos follow this structure.